Tribler/community/search/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+The search package contains TTTODO: what does this do?
```

Comment:
The search package contains a Dispersy community which is used to implement decentralized search in Tribler. It also allows peers to discover new .torrents.